homeassistant/components/isy994/sensor.py

```diff
@@ -47,16 +59,16 @@
     "37": "mercalli",
     "38": "m",
     "39": "m³/hr",
-    "40": "m/s",
+    "40": f"m/{TIME_SECONDS}",
     "41": "mA",
-    "42": "ms",
+    "42": TIME_MILLISECONDS,
     "43": "mV",
-    "44": "min",
-    "45": "min",
+    "44": TIME_MINUTES,
+    "45": TIME_MINUTES,
     "46": "mm/hr",
-    "47": "month",
+    "47": TIME_MONTHS,
     "48": "MPH",
-    "49": "m/s",
+    "49": f"m/{TIME_SECONDS}",
     "50": "ohm",
     "51": "%",
     "52": "lb",
```

Comment:
A couple of `"s"` below in 57 and 58, maybe `TIME_SECONDS` there? (GH won't let me add a comment to those lines.)

Reply:
Yeah I was not too sure if those were in seconds but I have added them now.